src/components/industry-use-cases/accordion-item-content/index.tsx

```diff
@@ -52,7 +53,156 @@ const AccordionItemContent = ({
+                {directIndustries.map(({ title, tooltip }) => (
+                  <li key={title} className="flex justify-between">
+                    {title}
+                    <InfoTooltip content={tooltip} />
+                  </li>
+                ))}
+              </ul>
+            </div>
+            <div className="flex min-w-full flex-col lg:min-w-[344px]">
+              <div className="mb-[10px] font-bold">Industry that indirectly relies on forests</div>
+              <ul className="flex flex-col gap-4 py-[10px]">
+                {indirectIndustries.map(({ title, tooltip }) => (
+                  <li key={title} className="flex justify-between">
+                    {title}
+                    <InfoTooltip content={tooltip} />
+                  </li>
+                ))}
+              </ul>
+            </div>
+          </div>
+        </div>
+        <Image
+          className="min-h-[300px] w-full object-cover"
+          src={image3}
+          alt=""
+          width={871}
+          height={300}
+        />
+      </div>
+      <div className="mb-10 flex flex-grow flex-col gap-10 px-6 lg:mb-0 lg:px-0">
+        <div className="flex items-center gap-3">
+          <h2>Key insights</h2>
+          <div className="grow border-t border-t-grey-500"></div>
+        </div>
+        <ol className="flex max-w-[540px] list-decimal-leading-zero flex-col gap-y-6 pl-9 lg:gap-y-5">
+          {insights.map((insight) => (
+            <li key={insight}>{insight}</li>
+          ))}
+        </ol>
+      </div>
+    </TabsContent>
+  );
+};
+
+const DependenciesTabContent = ({ data }: { data: AccordionItemContentType["dependencies"] }) => {
+  const { content1 } = data;
+
+  return (
+    <TabsContent value="dependencies" className="flex flex-col gap-y-10 text-black">
+      <div className="mt-5 flex flex-col gap-6 px-6 lg:gap-5 lg:px-0">{content1}</div>
+      {/* WIP image and text. Replace with chart */}
+      <div className="relative w-full">
+        <Image
+          className="mx-6 min-h-[388px] w-[calc(100%-48px)] max-lg:object-contain lg:mx-0 lg:w-full"
+          src="/assets/industry-cases-construction-dependencies-wip.png"
+          alt=""
+          width={830}
+          height={583}
+        />
+        <div className="absolute inset-0 left-12 top-1/2 h-fit w-full -rotate-12 text-3xl">
+          Interactive chart coming soon
+        </div>
+      </div>
+    </TabsContent>
+  );
+};
+
+const ImpactsTabContent = ({ data }: { data: AccordionItemContentType["impacts"] }) => {
+  const { content1, content2, image1, list } = data;
+  return (
+    <TabsContent value="impacts" className="flex flex-col gap-y-10 text-black">
+      <div className="mt-5 flex flex-col gap-6 px-6 lg:gap-5 lg:px-0">{content1}</div>
+      <div className="mx-6 lg:mx-0">
+        <ul className="flex flex-col gap-6 bg-orange px-6 py-[40px] lg:gap-20 lg:p-[50px]">
+          {list.map(({ title, text }, index) => (
```

Comment:
I wonder if the text shouldn't be animated as on the Key Concepts page. We have the same kind of list.